homeassistant/components/pegel_online/sensor.py

```diff
@@ -0,0 +1,93 @@
+"""PEGELONLINE sensor entities."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from aiopegelonline import Station
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+from .entity import PegelOnlineEntity
+from .model import PegelOnlineData
+
+
+@dataclass
+class PegelOnlineRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    fn_native_unit: Callable[[PegelOnlineData], str]
+    fn_native_value: Callable[[PegelOnlineData], StateType]
+
+
+@dataclass
+class PegelOnlineSensorEntityDescription(
+    SensorEntityDescription, PegelOnlineRequiredKeysMixin
+):
+    """PEGELONLINE sensor entity description."""
+
+
+SENSORS: tuple[PegelOnlineSensorEntityDescription, ...] = (
+    PegelOnlineSensorEntityDescription(
+        key="current_measurement",
+        translation_key="current_measurement",
+        state_class=SensorStateClass.MEASUREMENT,
+        fn_native_unit=lambda data: data["current_measurement"].uom,
+        fn_native_value=lambda data: data["current_measurement"].value,
+        icon="mdi:waves-arrow-up",
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the PEGELONLINE sensor."""
+    (coordinator, station) = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        [
+            PegelOnlineSensor(coordinator, station, description)
+            for description in SENSORS
+        ]
+    )
+
+
+class PegelOnlineSensor(PegelOnlineEntity, SensorEntity):
+    """Representation of a PEGELONLINE sensor."""
+
+    entity_description: PegelOnlineSensorEntityDescription
+
+    def __init__(
+        self,
+        coordinator: DataUpdateCoordinator,
+        station: Station,
+        description: PegelOnlineSensorEntityDescription,
+    ) -> None:
+        """Initialize a PEGELONLINE sensor."""
+        super().__init__(coordinator, station)
+        self.entity_description = description
+        self._attr_unique_id = f"{station.uuid}/{description.key}"
```

Comment:
The underscore is more the defacto standard.

```suggestion
        self._attr_unique_id = f"{station.uuid}_{description.key}"
```